FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -438,33 +455,42 @@ private DbDataReader ExecuteTest(DbConnection conn, string commandText, object e
         }
 
         [Test]
+        [Category("general")]
         public void ExecuteSelectByteA()
         {
             ExecuteTest(SYSTEM_CONNECTION_STRING, "SELECT 'hello_world_123ツ\n\u0048'::bytea", Encoding.UTF8.GetBytes("hello_world_123ツ\n\u0048"), typeof(byte[]));
         }
 
         [Test]
+        [Category("general")]
         public void ShouldThrowExceptionWhenEngineIsNotFound()
         {
             var connString = $"{SYSTEM_CONNECTION_STRING};engine=InexistantEngine";
             using var conn = new FireboltConnection(connString);
             FireboltException? exception = Assert.Throws<FireboltException>(() => conn.Open());
-            Assert.That(exception!.Message, Is.EqualTo($"Engine InexistantEngine not found."));
+            switch (conn.InfraVersion)
+            {
+                case 1: Assert.That(exception!.Message, Is.EqualTo("Engine InexistantEngine not found.")); break;
+                case 2: Assert.That(exception!.Response?.Trim(), Is.EqualTo("Engine 'InexistantEngine' does not exist, it is stopped or you don't have permission to access it")); break;
+            }
         }
```

Comment:
Same answer as previous. Even more. The purpose of this test is to connect to engine that does not exist. How I can be sure that I fail to connect because engine does not exist and not because the provided client_secret is wrong?

Reply:
The secret being wrong will return an unauthorised error code so that's probably not a concern. As in the previous discussion, let's make this a little less brittle by relaxing the string check.

Reply:
replaced by pattern matching